info-commands.go

```diff
@@ -369,16 +369,21 @@ type ErasureBackend struct {
 
 // ServerProperties holds server information
 type ServerProperties struct {
-	State      string            `json:"state,omitempty"`
-	Endpoint   string            `json:"endpoint,omitempty"`
-	Scheme     string            `json:"scheme,omitempty"`
-	Uptime     int64             `json:"uptime,omitempty"`
-	Version    string            `json:"version,omitempty"`
-	CommitID   string            `json:"commitID,omitempty"`
-	Network    map[string]string `json:"network,omitempty"`
-	Disks      []Disk            `json:"drives,omitempty"`
-	PoolNumber int               `json:"poolNumber,omitempty"`
-	MemStats   MemStats          `json:"mem_stats"`
+	State          string            `json:"state,omitempty"`
+	Endpoint       string            `json:"endpoint,omitempty"`
+	Scheme         string            `json:"scheme,omitempty"`
+	Uptime         int64             `json:"uptime,omitempty"`
+	Version        string            `json:"version,omitempty"`
+	CommitID       string            `json:"commitID,omitempty"`
+	Network        map[string]string `json:"network,omitempty"`
+	Disks          []Disk            `json:"drives,omitempty"`
+	PoolNumber     int               `json:"poolNumber,omitempty"`
+	MemStats       MemStats          `json:"mem_stats"`
+	GOMAXPROCS     int               `json:"gomaxprocs,omitempty"`
```

Comment:
Maybe GoMaxProcs

Reply:
Yea, I'll change the [existing](https://github.com/minio/madmin-go/blob/d90007c2e5453efaae7862a96dcd3f5fa21981d8/health.go#L801) as well.